src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -935,5 +935,31 @@ private static ErrorValue GetDiv0Error(IRContext irContext)
                 Message = "Division by zero"
             });
         }
+
+        private static FormulaValue Dec2Hex(IRContext irContext, NumberValue[] args)
+        {
+            var number = Math.Floor(args[0].Value);
+            var places = Math.Floor(args[1].Value);
+            if (double.IsInfinity(number) || double.IsInfinity(places))
+            {
+                return CommonErrors.OverflowError(irContext);
+            }
+
+            var result = Math.Floor(number).ToString("X").PadLeft((int)places, '0');
+        
+            return new StringValue(irContext, result);
+        }
+
+        private static FormulaValue Hex2Dec(IRContext irContext, StringValue[] args)
+        {
+            var number = args[0].Value;
+            var result = double.Parse(number, System.Globalization.NumberStyles.HexNumber);
```

Comment:
We may have helpers in StandardError that will do these check for you. (see other math functions for example)

Reply:
Not for this specific example - we are parsing a string as a _hexadecimal_ number.